app/assets/stylesheets/style_guide/modules/_all_mars.scss

```diff
@@ -0,0 +1,8 @@
+@import "grid_demo";
+@import "hero";
+@import "hero_simple";
+@import "partial";
+@import "partial-rendered";
+@import "personas";
+@import "section-header";
+@import "source_toggle";
```

Comment:
Now that we are specifically calling a all_mars module file, we can filter out the scss files that we aren't using. Definitely not grid_demo and I don't think we are using any hero css either? I bet we don't need many of those files.